spec/features/capybara_homepage_spec.rb

```diff
@@ -18,14 +18,17 @@
   after { Warden.test_reset! }
   it 'validates presence of header and footer on homepage' do
     visit 'http://0.0.0.0:3000'
+    # The theme at U of A doesn't match the upstream theme
```

Comment:
What does this mean? Does these tests fail because they don't match the upstream theme?

Reply:
Note for when merging upstream changes. Tests modified because the UofA UAL theme differs from the upstream theme and these tests were changed to conform the UAL theme